donation_base/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
@nikul-serpentcs  No need coding: utf-8 for python3/Odoo V11

Reply:
@YogeshMahera-SerpentCS Yes, but in Check [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/models/__init__.py#L1)

Reply:
it's universal template(v8,v9,v10,v11), So we can not remove from template.

but from python3 onwards encoding is already added. so we don't need more to define!

Reply:
if you still not-getting look at this conversion https://github.com/OCA/maintainer-tools/pull/328#issue-169684858